src/Symfony/Bundle/TwigBundle/Resources/config/console.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bridge\Twig\Command\DebugCommand;
+use Symfony\Bundle\TwigBundle\Command\LintCommand;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+
+    ->set('twig.command.debug', DebugCommand::class)
+    ->args([
+        ref('twig'),
+        param('kernel.project_dir'),
+        param('kernel.bundles_metadata'),
+        param('twig.default_path'),
+        ref('debug.file_link_formatter')->nullOnInvalid(),
+    ])
+    ->tag('console.command', ['command' => 'debug:twig'])
+
+    ->set('twig.command.lint', LintCommand::class)
+    ->args([ref('twig')])
+    ->tag('console.command', ['command' => 'lint:twig'])
+    ;
```

Comment:
I would like to propose a slightly different CS (which is also the one used in all PHP examples in the docs):

```suggestion
    $services = $container->services();

    $services
        ->set('twig.command.debug', DebugCommand::class)
        ->args([
            ref('twig'),
            param('kernel.project_dir'),
            param('kernel.bundles_metadata'),
            param('twig.default_path'),
            ref('debug.file_link_formatter')->nullOnInvalid(),
        ])
        ->tag('console.command', ['command' => 'debug:twig'])
    ;

    $services
        ->set('twig.command.lint', LintCommand::class)
        ->args([ref('twig')])
        ->tag('console.command', ['command' => 'lint:twig'])
    ;
```

I think a specific chain per services makes it easier to read which methods belong to one service configuration.

Reply:
Here's a different proposal ... if you indent the chained methods below each service, the config is probably more readable and you don't have to repeat the `$services` variable everywhere. Here's an example: https://github.com/EasyCorp/EasyAdminBundle/blob/ee6966a092b7377d5011e7402127e3b70fcfa109/src/Resources/config/services.php#L80

Reply:
I'm trying to limit indentationwhen not necessary. Having a blank line between services seems enough to me. EasyAdminBundle looks like too much indented to me and thus less clear.

Reply:
Ugh. This is obviously 100% subjective. But it also makes sense to me to indent more - to have the config for a service appear “under it” with an extra indentation.

Reply:
Change of style should be discussed within a team or publicly. After that happens, it should be changed everywhere, including documentation and stick to it. Until that happens, I agree that it should be consistent with existing style. But as for me, style in this PR looks really awkward.

Reply:
CS updated